test/e2e/kubectl/kubectl.go

```diff
@@ -486,17 +486,37 @@ var _ = SIGDescribe("Kubectl client", func() {
 			framework.ExpectEqual(ok, true)
 			framework.ExpectEqual(ee.ExitStatus(), 42)
 
+			serverVersion, _ := f.ClientSet.Discovery().ServerVersion()
+			serverMajorVersion, _ := strconv.Atoi(serverVersion.Major)
```

Comment:
How about using something like this: https://github.com/kubernetes/kubernetes/blob/release-1.18/test/e2e/framework/skipper/skipper.go#L246?